rfc/php_technical_committee.wiki

```diff
@@ -0,0 +1,180 @@
+  * dev branch - php-src branch consisting of PHP- prefix and major and minor version number (e.g. PHP-8.2).
+  * maintainer - person responsible for core subsystem (e.g. extension or SAPI) that is listed in the php-src EXTENSION file or the php-src OWNERS files if added in the future.
+  * issue - php-src GitHub issue, including pull requests.
+  * process changes - Any changes impacting the way how PHP project is managed.
+  * pull request - php-src GitHub pull request.
+  * RM - PHP release managers.
+  * user facing changes - Changes in PHP language and its semantics, user visible functions, classes, interfaces, constants and other user visible constructs.
+  * technical change - Any change to php-src that has no user-facing change, including the implementation details of an RFC-accepted user facing change.
+  * technical aspect - The portions of  pull request or commit in php-src or commit to php-src master and dev branches that would be technical changes.
+  * technical decision - Decision on a technical aspect.
+  * TC - PHP Technical Committee.
+
+To further clarify some of the defined terms, following subsection provides some useful examples.
+
+=== Changes ===
+
+It is important to clarify difference between user facing change and technical changes. As it is defined, the technical
+change is any php-src change that is not a user facing change. However, the implementation of user facing change is
+still a technical change. The meaning of that is that the part of the user facing change that defines the user
+interface is not a technical change.
+
+For example if there is a proposal to add accessors for PHP classes, then everything that is visible to user space
+like the constructs and their usage in PHP code is a user facing change. The actual C implementation and tests are
+a technical change.
+
+==== TC membership ====
+
+The TC consists of five elected members.  Any core developer is eligible to run for the TC.  One member of the TC,
+selected by the TC members themselves, is the TC Secretary.
+
+The TC Secretary is responsible for:
+
+  * Administering the election of the next set of TC members.
+  * Documenting and communicating any decisions of the TC.
+  * In case of a tie, the Secretary's vote will be considered the winning position.
+
+=== Elections ===
+
+The election process takes place annually, concurrent with the election of the Release Managers in approximately
+March/April.  (The exact date may vary year to year.)  The sitting TC Secretary officially begins the process with
+a call for nominations.  The nomination period lasts for one week.
+
+During that week, any eligible core developer may self-nominate by posting to the Internals mailing list.  If the
+candidate is paid to work on php-src or PHP extensions, they must state in their nomination which company or organization
+they work for.  Failure to do so will result in disqualification, as determined by the TC Secretary.
```

Comment:
I think we need to detail _why_ this should be disqualifying.

Reply:
I added `to prevent conflict of interest` so hopefully it is clearer. The intention is basically to make sure that users are aware that multiple TC members are from the same company that could potential impact a PHP core development.

Reply:
I will mark this as resolved. If you feel, it needs further clarification, please add a comment or suggestion.